src/tempo_cluster.py

```diff
@@ -22,6 +22,8 @@
 from ops import EventSource, Object, ObjectEvents
 from pydantic import BaseModel, ConfigDict
 
+from tempo_config import TempoBaseConfig
```

Comment:
I'd rename this to `TempoConfig`. The name 'base' usually indicates you're meant to subclass it or specialize it somehow.